app/views/users/registrations/edit.html.erb

```diff
@@ -1,27 +1,52 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
-
-<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= f.error_notification %>
-
-  <div class="form-inputs">
-    <%= f.input :email, required: true, autofocus: true %>
-
-    <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
-      <p>Currently waiting confirmation for: <%= resource.unconfirmed_email %></p>
+<div class="row">    
+  <div class="col-md-6">  
+    <h2>Edit Profile</h2>
+    
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+      <%= devise_error_messages! %>
+      
+      <div class="form-group">
+        <%= f.label :first_name %><br />
+        <%= f.text_field :first_name, autofocus: true, class: "form-control", placeholder: "Update first name" %>
```

Comment:
@ianroberts131 autofocus option is intended to be used on one form field per page. When the page loads, the cursor will focus in the form field that has the autofocus option. Keep the autofocus option in the first field here, and remove the autofocus option from the other form fields in this template.